### DISCUSSION TITLE
Review and merge the Pull Request

### DISCUSSION THREAD

**Discussion by vivekvardhan2810** [Akshat111111/Hedging-of-Financial-Derivatives#992] [General]
@Akshat111111 sir, please review and Merge my Pull request. I am asking since 1 week. 

Nestle India Stock price prediction

Issue number  #770 

PR number #844 

Please sir

**Comment by Akshat111111** [comment]
yeah

**Comment by vivekvardhan2810** [comment]
@Akshat111111 i have reopened the pr and issue pls review and merge it

**Reply by Akshat111111** [reply]
okay

**Comment by vivekvardhan2810** [comment]
I am closing this discussion 

Thank you Akshat sharma brother for merging

**Reply by Akshat111111** [reply]
yeah
